boards/xtensa/waveshare_esp32s3_touch_lcd_1_28/Kconfig.defconfig

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2024 Joel Guittet
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_WAVESHARE_ESP32S3_TOUCH_LCD_1_28
+
+config BOARD
+	default "waveshare_esp32s3_touch_lcd_1_28"
+
+config HEAP_MEM_POOL_SIZE
+	int
+	default 98304 if WIFI
+	default 40960 if BT
+	default 4096
+
+choice BT_HCI_BUS_TYPE
+	default BT_ESP32 if BT
+endchoice
+
+endif
+
+config ENTROPY_GENERATOR
+	default y
+
+config KERNEL_MEM_POOL
+	default y
+
+config INPUT
+	default y
```

Comment:
"Unless explicitly recommended otherwise by this section, leave peripherals and their drivers disabled by default." https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#general-recommendations

Reply:
Sure, done!